[2025-10-11T06:22:38.206+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-11T06:22:38.249+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pipeline_flow.check_create_stg_tables scheduled__1999-12-12T00:00:00+00:00 [queued]>
[2025-10-11T06:22:38.263+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pipeline_flow.check_create_stg_tables scheduled__1999-12-12T00:00:00+00:00 [queued]>
[2025-10-11T06:22:38.264+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-10-11T06:22:38.281+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): check_create_stg_tables> on 1999-12-12 00:00:00+00:00
[2025-10-11T06:22:38.294+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62 DeprecationWarning: This process (pid=12065) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-10-11T06:22:38.296+0000] {standard_task_runner.py:64} INFO - Started process 12070 to run task
[2025-10-11T06:22:38.295+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'pipeline_flow', 'check_create_stg_tables', 'scheduled__1999-12-12T00:00:00+00:00', '--job-id', '1308', '--raw', '--subdir', 'DAGS_FOLDER/pipeline_dag.py', '--cfg-path', '/tmp/tmphzw0l2pn']
[2025-10-11T06:22:38.298+0000] {standard_task_runner.py:91} INFO - Job 1308: Subtask check_create_stg_tables
[2025-10-11T06:22:38.372+0000] {task_command.py:426} INFO - Running <TaskInstance: pipeline_flow.check_create_stg_tables scheduled__1999-12-12T00:00:00+00:00 [running]> on host 321597a50810
[2025-10-11T06:22:38.509+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='oluwaseyiogundimu10@gmail.com' AIRFLOW_CTX_DAG_OWNER='seyi' AIRFLOW_CTX_DAG_ID='pipeline_flow' AIRFLOW_CTX_TASK_ID='check_create_stg_tables' AIRFLOW_CTX_EXECUTION_DATE='1999-12-12T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__1999-12-12T00:00:00+00:00'
[2025-10-11T06:22:38.511+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-11T06:22:38.533+0000] {connection.py:22} INFO - Successfully connected to Postgres.
[2025-10-11T06:22:38.621+0000] {connection.py:43} INFO - Table 'stg_users' in Postgres.
[2025-10-11T06:22:38.623+0000] {connection.py:22} INFO - Successfully connected to Postgres.
[2025-10-11T06:22:38.708+0000] {connection.py:43} INFO - Table 'users' in Postgres.
[2025-10-11T06:22:38.710+0000] {connection.py:22} INFO - Successfully connected to Postgres.
[2025-10-11T06:22:38.792+0000] {connection.py:43} INFO - Table 'stg_movies' in Postgres.
[2025-10-11T06:22:38.793+0000] {connection.py:22} INFO - Successfully connected to Postgres.
[2025-10-11T06:22:38.882+0000] {connection.py:43} INFO - Table 'movies' in Postgres.
[2025-10-11T06:22:38.883+0000] {connection.py:22} INFO - Successfully connected to Postgres.
[2025-10-11T06:22:39.018+0000] {connection.py:43} INFO - Table 'stg_ratings' in Postgres.
[2025-10-11T06:22:39.020+0000] {connection.py:22} INFO - Successfully connected to Postgres.
[2025-10-11T06:22:39.143+0000] {connection.py:43} INFO - Table 'ratings' in Postgres.
[2025-10-11T06:22:39.144+0000] {load.py:32} INFO - Tables exist.
[2025-10-11T06:22:39.146+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-10-11T06:22:39.147+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-11T06:22:39.169+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=pipeline_flow, task_id=check_create_stg_tables, run_id=scheduled__1999-12-12T00:00:00+00:00, execution_date=19991212T000000, start_date=20251011T062238, end_date=20251011T062239
[2025-10-11T06:22:39.435+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-10-11T06:22:39.492+0000] {taskinstance.py:3503} INFO - 3 downstream tasks scheduled from follow-on schedule check
[2025-10-11T06:22:39.495+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
